docs/en/basic/Command-Line-Interface.md

```diff
@@ -586,105 +581,107 @@ For example, `ls` can be used to browse the file system.
+Using `mkdir` on an invalid or existing path will fail.
 
 For example, `mkdir` can be used by an admin to set up the basic folder structures.
 
 {% include Command-Line-Interface/mkdir.md %}
 
 ### mount
 
-The `mount` command links an under storage path to an Alluxio path, and files and folders created in
-Alluxio space under the path will be backed by a corresponding file or folder in the under storage
-path. For more details, see [Unified Namespace]({{site.baseurl}}{% link en/advanced/Namespace-Management.md %}).
+The `mount` command links an under storage path to an Alluxio path,
+where files and folders created in Alluxio space under the path will be backed
+by a corresponding file or folder in the under storage path.
+For more details, see [Unified Namespace]({{site.baseurl}}{% link en/advanced/Namespace-Management.md %}).
 
 Options:
 
-* `--readonly` option sets the  mount point to be readonly in Alluxio
-* `--option <key>=<val>` option passes an property to this mount point (e.g., S3 credential)
+* `--readonly` option sets the mount point to be readonly in Alluxio
+* `--option <key>=<val>` option passes an property to this mount point, such as S3 credentials
 
-Note that `--readonly` mounts are useful to prevent accidental write operations. If multiple 
-Alluxio satellite clusters mount a remote storage cluster which serves as the central source of
-truth, `--readonly` option could help prevent any write operations, e.g. rm -R, on the satellite
-cluster  from wiping out the remote storage.
+Note that `--readonly` mounts are useful to prevent accidental write operations.
+If multiple Alluxio satellite clusters mount a remote storage cluster which serves as the central source of truth,
+`--readonly` option could help prevent any write operations on the satellite cluster from wiping out the remote storage.
 
 For example, `mount` can be used to make data in another storage system available in Alluxio.
 
 {% include Command-Line-Interface/mount.md %}
 
 ### mv
 
-The `mv` command moves a file or directory to another path in Alluxio. The destination path must not
-exist or be a directory. If it is a directory, the file or directory will be placed as a child of
-the directory. `mv` is purely a metadata operation and does not affect the data blocks of the file.
+The `mv` command moves a file or directory to another path in Alluxio.
+The destination path must not exist or be a directory.
+If it is a directory, the file or directory will be placed as a child of the directory.
+`mv` is purely a metadata operation and does not affect the data blocks of the file.
 `mv` cannot be done between mount points of different under storage systems.
 
-For example, `mv` can be used to move older data into a non working directory.
+For example, `mv` can be used to move older data into a non-working directory.
+TODO: huh? what's a non-working directory? how does working directory even relate to this???
```

Comment:
-For example, `mv` can be used to move older data into a non-working directory.
-TODO: huh? what's a non-working directory? how does working directory even relate to this???
+For example, `mv` can be used to re-organize your files.